Resources/Prototypes/DeltaV/Entities/Objects/Devices/Syndicate_Gadgets/reinforcement_teleporter.yml

```diff
@@ -18,3 +18,20 @@
     sound: /Audio/Animals/mouse_squeak.ogg
   - type: UseDelay
     delay: 300
+
+- type: entity
+  parent: ReinforcementRadioSyndicate
```

Comment:
Somewhere in the parent hierarchy is a fart noise generator. Said noise generator should be overridden with something more serious by adding a matching component to this entity.